docs/source/conf.py

```diff
@@ -51,6 +52,10 @@
 # a list of builtin themes.
 #
 html_theme = 'sphinx_rtd_theme'
+html_logo = '_static/piq_logo.png'
```

Comment:
Option 1 (like it is done by pytorch):
1. Let's use logo with transparent background without "by Photosynthesis Team"
2. Let's use the light-grey or white background of the readthedocs theme.

Option 2:
Use just text

I tried to use different combinations of background/logo, unfortunately none of them were satisfying enough to use. Therefore, I suggest to keep just title of the library.